$MirOS: ports/lang/pawn/toolkit/patches/patch-source_amx_amxfile_c,v 1.1.1.1 2008/08/08 15:53:57 tg Exp $
--- source/amx/amxfile.c.orig	Thu Mar  6 13:16:05 2008
+++ source/amx/amxfile.c	Fri Aug  8 18:57:01 2008
@@ -44,7 +44,7 @@
 #if defined __WIN32__ || defined _Windows
   #include <windows.h>
 #endif
-#if defined __LINUX__ || defined __FreeBSD__ || defined __OpenBSD__ || defined MACOS
+#if defined __LINUX__ || defined BSD || defined MACOS
   #include <dirent.h>
 #else
   #include <io.h>
@@ -743,7 +743,7 @@ static int matchfiles(const TCHAR *path,
       strcpy(dirname,".");
     } else {
       strncpy(dirname,path,(int)(basename-path));
-      dirname[(int)(basename-path)]=_T('\0');
+      dirname[(int)(basename-path)]=__T('\0');
     } /* if */
     if ((dir=opendir(dirname))!=NULL) {
       while ((entry=readdir(dir))!=NULL) {
